sql2.txt

```diff
@@ -17,29 +42,86 @@ may have ordered the same product in several different orders.  So the total qua
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
 
+              - SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity
+                FROM Customers c
+                JOIN Orders o
+                ON c.CustomerID = o.CustomerID
+                JOIN OrderDetails od
+                ON o.OrderID = od.OrderID
+                JOIN Products p
+                ON od.ProductID = p.ProductID
+                GROUP BY c.CustomerID, p.ProductID;
+
+
+
+
+
 
 
 4. Extend the previous query to give the total price as well as the customer name, product name, 
 and total quantity.  Here you will have
 (SUM(Quantity) * Price) AS TotalPrice.  Paste your SQL statement below.
 
+              - SELECT c.CustomerName, p.ProductName, SUM(od.Quantity) AS TotalQuantity, (SUM(od.Quantity) * p.Price) AS TotalPrice
+                FROM Customers c
+                JOIN Orders o
+                ON c.CustomerID = o.CustomerID
+                JOIN OrderDetails od
+                ON o.OrderID = od.OrderID
+                JOIN Products p
+                ON od.ProductID = p.ProductID
+                GROUP BY c.CustomerID, p.ProductID;
+
+
+
+
+
 
 5. Add a new Order to the Orders table.  You will have to specify the ID of an existing customer
 to satisfy the foreign key constraint.  You will also have to specify the ID of an existing
 employee.  Paste your SQL statement below.
 
+              - INSERT INTO Orders (CustomerID, EmployeeID, OrderDate, ShipperID)
+                VALUES (34, 3, "7/4/1996 ", 3);
+
+
+
+
+
+
 
 6. Add three OrderDetails.  These should specify the ID of the new Order you just created.  For each,
 you will have to specify the ID of an existing product.  Paste your SQL statement below.
 
+              - INSERT INTO OrderDetails (OrderID, ProductID, Quantity)
+                VALUES (10333, 5, 20), (10444,  6, 20), (10284, 3, 5);
+
+
+
```

Comment:
nice - but the order number should remain consistent - you need the same order number.